components/mail/dovecot/Makefile

```diff
@@ -57,9 +60,7 @@ CONFIGURE_OPTIONS += --with-mysql
 CONFIGURE_OPTIONS += --with-pgsql
 CONFIGURE_OPTIONS += --enable-static=no
 CONFIGURE_OPTIONS += --without-systemd
-CONFIGURE_OPTIONS += SSL_CFLAGS=-I$(OPENSSL_INCDIR)
-CONFIGURE_OPTIONS += SSL_LIBS="-L$(OPENSSL_LIBDIR) -lssl -lcrypto"
-CONFIGURE_OPTIONS += LDFLAGS="-lldap_r"
+CONFIGURE_OPTIONS += LDFLAGS="-lldap_r -licuuc"
```

Comment:
Please add a comment explaining why do we need `-licuuc` in `LDFLAGS`.

Reply:
will do